SETUP.md

```diff
@@ -0,0 +1,55 @@
+# Setup
+
+## Installing Docker
+
+### MacOS
+
+1. If not already installed, install [Homebrew](https://brew.sh/), a package
+   manager for MacOS and Linux
+2. Install Docker: `brew install --cask docker`
+3. Open the newly installed _Docker Desktop_ application
+   1. You should be able to find it in the application folder
+   2. Otherwise, use Spotlight search (<kbd>⌘</kbd> + <kbd>space</kbd>) and
+      search for "Docker"
+4. Select "Use recommended settings"
+5. Skip remaining steps if possible
+
+### Linux
+
+1. See [Docker install documentation](https://docs.docker.com/engine/install/)
+   (untested)
+
+### Windows
+
+1. See [Docker install documentation](https://docs.docker.com/engine/install/)
+   (untested)
+
+## Running Docker
+
+1. Set your working directory to the top level of the Q-learning repository
+2. To run a command, use `docker-compose run nav --rm [COMMANDS]`
+   1. This command will create an image and container (if necessary) and execute
+      the given commands
+   2. The `--rm` option deletes the container that was created to run the
```

Comment:
I found that my installation didn't work when running with the --rm flag but worked otherwise